docs/Makefile

```diff
@@ -0,0 +1,192 @@
+# Makefile for Sphinx documentation
```

Comment:
I wonder if it makes sense to have such a verbose config. For example, I find it highly unlikely that anyone would ever want to compile a qthelp version. But I guess that's a more general question to discuss with @evildmp .